### ISSUE TITLE
Issue with errors

### ISSUE THREAD

**Issue by JeffreyArts** [lunelson/penny#15]
Hi,

Just starting experimenting with this repo which I saw you mention in some harpjs issue. Really like it so far, since it has that what I am looking for in a new project for experiments (just sass, es6 support & an HTTP server). But every time an error occurs in the js. The server crashes. Which I then need to restart manually. How can I resolve this?

Error occurs both when I have .eslintrc.json file or without it.

```
/.nvm/versions/node/v7.9.0/lib/node_modules/@lunelson/penny/node_modules/eslint-loader/index.js:79
var messages = config.formatter(res.results)
                            ^

TypeError: config.formatter is not a function
    at printLinterOutput (/Users/jeffreyarts/.nvm/versions/node/v7.9.0/lib/node_modules/@lunelson/penny/node_modules/eslint-loader/index.js:79:29)
    at /Users/jeffreyarts/.nvm/versions/node/v7.9.0/lib/node_modules/@lunelson/penny/node_modules/eslint-loader/index.js:208:9
    at /Users/jeffreyarts/.nvm/versions/node/v7.9.0/lib/node_modules/@lunelson/penny/node_modules/loader-fs-cache/index.js:139:9
    at /Users/jeffreyarts/.nvm/versions/node/v7.9.0/lib/node_modules/@lunelson/penny/node_modules/graceful-fs/graceful-fs.js:43:10
    at FSReqWrap.oncomplete (fs.js:114:15)
```

**Comment by lunelson** [comment]
Hi, I'm glad to hear you find this useful. It's a bit of a work in progress still:`eslintrc` files aren't being picked up right now, it's just a minimal set of `eslint` rules to flag things that are likely to produce errors, and they are hard-coded in. But yes I've seen this error and might have to disable eslint until I find time to trace it. I'll see if I can make a patch release in the next day or so to either sort it or suppress it.

**Comment by JeffreyArts** [comment]
I am not sure if I understood your message correctly. 

Are you saying that the error caused by a set of (hard-coded) eslint rules? 

I have been using [this](https://github.com/piotr-gawlowski/ng-webpack-seed) skeleton for angular set-ups which also uses es linting and looks very similar as far as I can tell. Maybe this could give you some insights in helping resolving the issue?

**Comment by lunelson** [comment]
Yes, the rules are hard-coded right now, just a very reduced set taken from those that eslint recommends as "errors" i.e. "things that are probably typos and will cause your code to fail". Nothing opinionated in terms of coding style. My intention is to allow either an `eslintrc` file or an `eslintRules` option in a `pennyrc` file, which would override those rules. However this is not yet implemented.

**Comment by JeffreyArts** [comment]
Okay... But my problem is _not_ that I can not use a custom lint file. My problem is that instead of the compiler logging the error (which is what the linter is supposed to do). The compiler just crashes.

**Comment by lunelson** [comment]
Yeah I know. It's something wrong with the way I've configured it I think. I'll cut it out for now probably. If you can isolate an example of a JS file that causes this crash though, that would be helpful.

**Comment by lunelson** [comment]
I published 0.8.1 to npm, with the `formatter` option in eslint disabled, which seems to fix it. It still gets some weird errors but it doesn't crash and eslint warnings come through, even if formatting is not very good right now

**Comment by lunelson** [comment]
I've just published `1.0.0` of this package with fixes to eslint usage, that should cover the errors you were having. The eslint config is not customizable yet but covers errors and import failures. This version also corrects some webpack compiling problems
